src/email/html/email-verification.ftl

```diff
@@ -0,0 +1,16 @@
+<#import "template.ftl" as layout>
+<@layout.emailLayout>
+    <p>${msg("emailVerificationBodyIntro")}</p>
+
+    <p><a href="${link}">${msg("emailVerificationLinkText")}</a></p>
+
+    <p>${msg("emailVerificationExpiry", linkExpirationFormatter(linkExpiration))}</p>
+
+    <p>${msg("emailVerificationIgnore")}</p>
+
```

Comment:
Just one small change `FAIMER - A Division of Intealth`
Also can we change the email font to `Roboto`
 
![Screenshot 2024-12-17 at 3 49 15 PM](https://github.com/user-attachments/assets/49abc54a-12cc-4785-a860-c13d94a25ca4)